app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -323,6 +354,20 @@ public static void setScreenBrightness(@NonNull final Context context,
         setScreenBrightness(context, setScreenBrightness, System.currentTimeMillis());
     }
 
+    public static boolean globalScreenOrientationLocked(final Context context) {
+        // 1: Screen orientation changes using accelerometer
+        // 0: Screen orientation is locked
+        return android.provider.Settings.System.getInt(
+                context.getContentResolver(), Settings.System.ACCELEROMETER_ROTATION, 0) == 0;
+    }
+
+    public static boolean isTablet(@NonNull final Context context) {
+        return (context
+                .getResources()
+                .getConfiguration().screenLayout & Configuration.SCREENLAYOUT_SIZE_MASK)
+                >= Configuration.SCREENLAYOUT_SIZE_LARGE;
+    }
```

Comment:
This should be moved into a util class. Please rename [AndroidTvUtils](https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/java/org/schabi/newpipe/util/AndroidTvUtils.java) to DeviceUtils and add it there.